src/plugin/plugin.ts

```diff
@@ -582,6 +582,12 @@ export interface PurchasesConfiguration {
    * suite, otherwise it will use standardUserDefaults. Default is null, which will make the SDK use standardUserDefaults.
    */
   userDefaultsSuiteName?: string;
+  /**
+   * iOS-only, will be ignored for Android.
+   * Set this to FALSE to disable StoreKit2.
+   * Default is TRUE.
```

Comment:
Default is `false` now, right?

Reply:
oh yes. Good catch. I wonder where I got this from, but we probably need to update it there too. I will look for it